feature/bgp/multipath/otg_tests/bgp_multipath_wecmp_lbw_community_test/README.md

```diff
@@ -0,0 +1,69 @@
+# RT-1.52: BGP multipath ECMP support with Link Bandwidth Community
+
+## Summary
+
+Validate BGP in multipath ECMP support with link bandwidth community
+
+## Testbed type
+
+[TESTBED_DUT_ATE_4LINKS](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_4.testbed)
+
+## Procedure
+
+### Setup
+
+*   Connect DUT port 1, 2 and 3 to ATE port 1, 2 and 3 respectively
+*   Configure IPv4/IPv6 addresses on the interfaces
+*   Establish eBGP sessions between:
+    *   ATE port-1 and DUT port-1
+    *   ATE port-2 and DUT port-2
+    *   ATE port-3 and DUT port-3
+*   Enable an Accept-route all import-policy/export-policy for eBGP session
+    under the neighbor AFI/SAFI
+*   Create an IPv4 internal target network attached to ATE port 2 and 3
+
+### Tests
+
+*   RT-1.52.1: Verify use of community type
```

Comment:
We need to add something to make the bandwidths for port 2 and 3 unequal.  I suggest for this subtest, we require that one of the ports is 100G and the other port is 400G.  

We will need to add a subnet in the near future to configure link bandwidth, but this configuration is pending https://github.com/openconfig/public/pull/883